insights/client/connection.py

```diff
@@ -1086,19 +1119,16 @@ def checkin(self):
         url = self.inventory_url + "/hosts/checkin"
         logger.debug("Sending check-in request to %s with %s" % (url, canonical_facts))
         try:
-            response = self.session.post(url, headers={"Content-Type": "application/json"}, data=json.dumps(canonical_facts))
-            # Change to POST when the API is fixed.
-        except REQUEST_FAILED_EXCEPTIONS as exception:
-            _api_request_failed(exception)
+            response = self.post(url, headers={"Content-Type": "application/json"}, data=json.dumps(canonical_facts))
+        except RuntimeError as e:
+            logger.error(e)
```

Comment:
This is unnecessary. The call is [wrapped](https://github.com/RedHatInsights/insights-core/blob/9651f0c7e12e5e88ad29d3db1e74f119ce7fc9ad/insights/client/phase/v1.py#L115) by _except Exception_ that does the same logging.